### PR TITLE
fix(clustering) fix rpc id of negotiation

### PR DESCRIPTION
**Pull request by Suika-Kong**
Should we also change the id of the service? This service is more fundamental than config.

### CODE REVIEW

**Review by javierguerragiraldez — APPROVED** [review]
great!
with Kong/koko#344  we'll fix this at both sides

File: kong/include/kong/services/negotiation/v1/negotiation.proto [hunk 1]
```diff
@@ -12,7 +12,7 @@ service NegotiationService {
   // to get which services does the DP handle.
   //
   // Call direction: DP to CP
-  // +wrpc:rpc-id=4
+  // +wrpc:rpc-id=1
```
Comment (chronolaw) [inline kong/include/kong/services/negotiation/v1/negotiation.proto:15]: Could you give a short explain for this id? What is the number's principle?

**Review by chronolaw — APPROVED** [review]

File: kong/include/kong/services/negotiation/v1/negotiation.proto (hunk 1 above)
Comment (chronolaw) [reply kong/include/kong/services/negotiation/v1/negotiation.proto:15]: I think it is just to sync up with koko?

Comment (Suika-Kong) [reply kong/include/kong/services/negotiation/v1/negotiation.proto:15]: Yes. This was a typo, and koko already fix it.

**Comment by javierguerragiraldez** [comment]
friendly remainder that Kong/koko#344 is already merged and deployed.

**Pull request merged**